closed/make/Javadoc.gmk

```diff
@@ -81,20 +81,20 @@ OPENJ9_SHAREDCLASSES_INDEX_FILE := $(OPENJ9_SHAREDCLASSES_DOCDIR)/index.html
 OPENJ9_SHAREDCLASSES_OPTIONS_FILE := $(DOCSTMPDIR)/openj9.sharedclasses.options
 OPENJ9_SHAREDCLASSES_PACKAGES_FILE := $(DOCSTMPDIR)/openj9.sharedclasses.packages
 
-openj9-sharedclassesdocs: $(OPENJ9_SHAREDCLASSES_INDEX_FILE)
+openj9-sharedclassesdocs : $(OPENJ9_SHAREDCLASSES_INDEX_FILE)
 
 # Set relative location to platform document root
 $(OPENJ9_SHAREDCLASSES_INDEX_FILE): GET2DOCSDIR := $(PLATFORM_DOCSDIR)/..
```

Comment:
Space around `:` - here, and lines 280 & 560. Also, the additions to `closed/make/Main.gmk`.

Reply:
Fixed